service/src/main/java/org/cbioportal/genome_nexus/service/internal/CachedVariantAnnotationFetcher.java

```diff
@@ -0,0 +1,24 @@
+package org.cbioportal.genome_nexus.service.internal;
+
+import org.cbioportal.genome_nexus.model.VariantAnnotation;
+import org.cbioportal.genome_nexus.persistence.VariantAnnotationRepository;
+import org.cbioportal.genome_nexus.persistence.internal.VariantAnnotationRepositoryImpl;
```

Comment:
why VariantAnnotationRepositoryImpl in internal vs VariantAnnotationRepository in persistence?